content/en/docs/tasks/observability/metrics/telemetry-api/index.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Customizing Istio Metrics with Telemetry API
+description: This task shows you how to customize the Istio metrics with Telemetry API.
+weight: 10
```

Comment:
This weight will place this item as first on the page (https://deploy-preview-12916--preliminary-istio.netlify.app/latest/docs/tasks/observability/metrics/). Maybe you want it after the Customizing Istio Metrics, so then would be 26?

Reply:
I want to put this page first, cause Telemetry API is the recommended way.